inc/event_queue.h

```diff
@@ -1,10 +1,12 @@
 #pragma once
 #include <stdint.h>
 #include <stdbool.h>
+#include "objpool.h"
```

Comment:
I mean, it doesn't really matter to me tbh. If it gets fixed automatically by the linter I'm okay with it.
We should probably define a rule for module documentation and where it should go.